src/pages/projects/i-love-you-mom-app/privacy-policy.js

```diff
@@ -0,0 +1,159 @@
+import React from "react"
+import { Link } from "gatsby"
+
+import Layout from "../../../components/layout"
+import SEO from "../../../components/seo"
+
+const IndexPage = () => (
+  <Layout>
+    <div>
+    <SEO title="I Love You Mom App" />
+    <h1>I Love You Mom &lt;3 Privacy Policy</h1>
+    <h2>PRIVACY NOTICE</h2>
+	<p>Last updated April 05, 2020</p>
+<p>Thank you for choosing to be part of our community at I Love You Mom &lt;3 (“Company”, “we”, “us”, or “our”). We are committed to protecting your personal information and your right to privacy. If you have any questions or concerns about our notice, or our practices with regards to your personal information, please contact us at __________.
+When you visit our mobile application, and use our services, you trust us with your personal information. We take your privacy very seriously. In this privacy notice, we seek to explain to you in the clearest way possible what information we collect, how we use it and what rights you have in relation to it. We hope you take some time to read through it carefully, as it is important. If there are any terms in this privacy notice that you do not agree with, please discontinue use of our Apps and our services.
+
+This privacy notice applies to all information collected through our mobile application, ("Apps"), and/or any related services, sales, marketing or events (we refer to them collectively in this privacy notice as the "Services").
+
+Please read this privacy notice carefully as it will help you make informed decisions about sharing your personal information with us.</p>
+
+
+<h3>Table of Contents</h3>
+<p style="font-size: 15px;"><a href="#infocollect"><span style="color: #595959;">1. WHAT INFORMATION DO WE COLLECT?</span></a></p>
+<p style="font-size: 15px;"><a href="#infoshare">2. WILL YOUR INFORMATION BE SHARED WITH ANYONE?</a></span></p>
```

Comment:
Here you just have a closing </span>

You can either remove it or add an opening <span> before the <a> tag